features/bookmarks/bookmark_external.feature

```diff
@@ -101,3 +101,45 @@ Feature: Create bookmarks of external works
     Then I should not see "Bookmark External Work"
     When I go to the bookmarks in collection "Testing BEW Collection"
     Then I should not see "Bookmark External Work"
+
+  @javascript
+  Scenario: When using the bookmarklet on a URL that has already been bookmarked, the work information will be automatically filled in
+    Given "first_user" has bookmarked an external work
+      And I am logged in as "second_user"
+    When I use the bookmarklet on a previously bookmarked URL
+      And I wait 1 second
```

Comment:
Does this work without the wait step and with #3812?

Reply:
No, it turns out I added the wait to give the JavaScript a chance to populate the other fields based on the selected URL, not because of a delay with the autocomplete menu appearing. 

I've put the wait in the `When I use the bookmarklet on a previously bookmarked URL` step definition now with a comment so we don't waste time trying to fix it later.